src/elements/pages/dev-dash.ts

```diff
@@ -0,0 +1,21 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { cssify } from '../../utils/css';
+import styles from './home.scss';
+
+@customElement('dev-dash')
+export default class DeveloperDash extends LitElement {
+	static styles = cssify(styles);
+
+	render() {
+		return html`
+			<div class="w-full">
+				<user-banner></user-banner>
+
+				<div class="w-[97%] max-w-[1100px] m-auto">
```

Comment:
Percentages are not good for building layouts. Unless trying to maintain an aspect ratio, you should always ben using constant padding/margins combined with media queries.

Reply:
Can we find (or define) a standard TW width for the 1100px? This seems like something that should be able to be standardized pretty easily.